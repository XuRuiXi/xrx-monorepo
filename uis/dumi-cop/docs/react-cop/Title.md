---
group: '显示组件'

title: 'Title'

toc: content
---

```jsx
/**
  title: 基础使用
  description: 标题组件
*/
import { Title } from 'hk-library';

export default () => <Title />;
```
