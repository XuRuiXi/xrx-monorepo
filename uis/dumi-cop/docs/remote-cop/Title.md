---
nav: '远程组件库'

group: '展示组件'

title: 'Title'

toc: content
---


```jsx | pure
/**
  title: 基础使用
  description: 海口各个项目通用标题组件
*/
// 引入js文件
<script src="./hk-library.min.js"></script>
// 在window暴露了一个Title的组件
const { Title, ReactDOM } = hkLibrary;
// 通过ReactDOM.render方法渲染组件
ReactDOM.render(Title({ title: '标题' }), targetElement);
```