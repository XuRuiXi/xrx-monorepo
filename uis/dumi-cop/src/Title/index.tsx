import React from 'react';

const Title = ({ title = '标题' }: { title: string }) => <div>{title}</div>;

export default Title;
