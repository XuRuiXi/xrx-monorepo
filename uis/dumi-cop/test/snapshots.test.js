import React from 'react';
import renderer from "react-test-renderer";
import Title from "../src/Title";

describe("Tabs", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<Title title="我是标题new" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
