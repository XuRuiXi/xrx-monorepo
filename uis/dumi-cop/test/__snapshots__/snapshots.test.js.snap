// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders correctly 1`] = `
<div>
  我是标题new
</div>
`;
